cmd/mcp-server/scenarios/component-misconfiguration-teardown.sh

```
@@ -0,0 +1,45 @@
+
+require_tools oc
+
+APPS_NS=$(discover_apps_namespace)
+
+log_step "=== Scenario: Component Misconfiguration — Teardown ==="
+
+STATE_DIR="${XDG_RUNTIME_DIR:-/tmp}/odh-mcp-scenarios-${UID}"
+
+# Kill the background loop
+if [[ -f ${STATE_DIR}/odh-misconfiguration-pid ]]; then
+  LOOP_PID=$(cat ${STATE_DIR}/odh-misconfiguration-pid)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)